src/index.jade

```diff
@@ -17,17 +18,17 @@ html
     ])
 
     header.py2.flex.flex-wrap.items-center
-      div.md-flex.flex-wrap.items-center.header-wrap
+      div.md-flex.flex-wrap.items-center.header-wrap.overflow-scroll
         div.col-10.md-col-8.right-align.mx-auto
           h1.m0 Nakano#[br.md-hide.lg-hide] Computer Club
           p.h3.mt1.mb1 NCC
         div.col-10.md-col-4.mx-auto
           //- no centering
           p.mx2.xs-hide.sm-hide
-            img(src="//kauntah-svg.mybluemix.net/counter.svg", height="100")
+            img(src="http://kauntah.herokuapp.com/counter?offset=0", height="100")
```

Comment:
This image will be served over HTTP, but `meiji-ncc.tech` serve contents over HTTPS.
Loading of [Mixed Content](https://web.dev/what-is-mixed-content/) will be blocked by browser.

Serve image over HTTPS as below:
```suggestion
            img(src="https://kauntah.herokuapp.com/counter?offset=0", height="100")
```